fractal/main.go

```diff
@@ -25,15 +26,15 @@ func (f *fractal) MinSize(objects []fyne.CanvasObject) fyne.Size {
 	return fyne.NewSize(320, 240)
 }
 
-func (f *fractal) refresh() {
-	if f.currScale >= 1.0 {
-		f.currIterations = 100
-	} else {
-		f.currIterations = uint(100 * (1 + math.Pow((math.Log10(1/f.currScale)), 1.25)))
-	}
-
-	f.window.Canvas().Refresh(f.canvas)
-}
+// func (f *fractal) refresh() {
```

Comment:
I accidentally marked this as resolved instead of letting you resolve it like we usually do. Either way, this is fixed now so there's little point in unresolving this :)